File: math/111-999-two-digits-no-zero/solve-arshia.py
def has_conditions(num):
    result = set() # creating a set
    for _ in range(3):
        digit = num % 10
        num = num // 10
        result.add(digit) # adding digits to set
    if 0 not in result and len(result) == 2: # check if we have 0 in our set and our set length
        return True
    return False
        
def main():
    count = 0
    for i in range(112, 1000):
        if has_conditions(i):
            count += 1
    print(count)
        

File: math/111-999-two-digits-no-zero/solve-arshia.js
function hasConditions(num) {
    const digitSet = new Set(); // creating a set
    for (i = 0; i < 3; i++) {
      let digit = num % 10;
      num = Math.floor(num / 10); // finding numbers
      digitSet.add(String(digit)); // adding str nums to a set
    }
    console.log(digitSet.size);
    if (digitSet.size !== 2 || digitSet.has("0")) {
      // check if we have 0 in our set and our set length
      return false;
    }
    return true;
  }
  

function main() {
  let count = 0;
  for (let i = 112; i < 1000; i++) {
    if (hasConditions(i)) {
      count++;
    }
  }
  console.log(count);
}


main(); // calling the main so we can see the result
